build/PackageArchive.targets

```diff
@@ -0,0 +1,78 @@
+<Project>
+  <PropertyGroup>
+    <_TemplatesDirectory>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDirectory>
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+  </PropertyGroup>
+
+  <Target Name="_BuildFallbackArchive" DependsOnTargets="ResolveRepoInfo">
+    <Error Text="DotNetRestoreSources must be specified" Condition=" '$(DotNetRestoreSources)' == '' " />
+    <Error Text="OutputPackageName must be specified" Condition=" '$(OutputPackageName)' == '' " />
+    <Error Text="ArchiverPath must be specified" Condition=" '$(ArchiverPath)' == '' " />
+    <Error Text="Archiver not found at $(ArchiverPath)" Condition="!Exists('$(ArchiverPath)')" />
+
+    <!-- Clear the directories -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+
+    <!-- Copy the archive template -->
+    <Copy SourceFiles="$(_TemplatesDirectory)Archive\Archive.csproj" DestinationFiles="$(_WorkRoot)Archive.csproj" />
+
+    <!-- Copy the archive template -->
+    <RepoTasks.AddArchiveReferences
+      ReferencePackagePath="$(_WorkRoot)Archive.csproj"
+      BuildArtifacts="@(ArtifactInfo)"
+      PackageArtifacts="@(PackageArtifact)"
+      ExternalDependencies="@(ExternalDependency)"
+      RemoveTimestamp="$(RemoveTimestamp)" />
+
+    <PropertyGroup>
+      <FallbackStagingDir>$(RepositoryRoot)obj\$(OutputPackageName)</FallbackStagingDir>
```

Comment:
FYI, KoreBuild defines $(IntermediateDir) to $(RepositoryRoot)obj as a default property you can use.

Reply:
actually this should be in the workdir.